db/migrate/20151023144501_regenerate_red_hat_kexec.rb

```diff
@@ -0,0 +1,10 @@
+class RegenerateRedHatKexec < ActiveRecord::Migration
+  def up
+    t = ProvisioningTemplate.find_by_name("Discovery Red Hat kexec")
+    t.update_attributes(:template => t.template.sub(/rescue ''$/, 'if mac'))
```

Comment:
I think you need `if t`

DB migration will run before first seed, so on a new install I think this will raise an error because t will be nil.